src/components/node-list/node-list-tree-search.js

```diff
@@ -0,0 +1,209 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import TreeView from '@material-ui/lab/TreeView';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+
```

Comment:
@studioswong  -- I would like to understand a bit more on why do we use material-ui icons and not the svg arrows from Gabriel's design like we generally do for the rest of the icons in kedro?

Reply:
the arrow used here from Gabriel's design is a generic  'expand more' icon ( the icon used in the tree is different from the other expand more icon used on the sidebar) - given that that icon is only used here, and since material-ui provides that out of the box, I opted for the simplest solution to use their icon. This also avoids the need to particularly maintain more code for a one-off icon just for the tree.